priv/cabbage/apps/itest/lib/account.ex

```diff
@@ -9,7 +9,11 @@ defmodule Itest.Account do
   @spec take_accounts(integer()) :: map()
   def take_accounts(number_of_accounts) do
     1..number_of_accounts
-    |> Task.async_stream(fn _ -> account() end, timeout: 60_000, on_timeout: :kill_task)
+    |> Task.async_stream(fn _ -> account() end,
+      timeout: 60_000,
+      on_timeout: :kill_task,
+      max_concurrency: System.schedulers_online() * 2
```

Comment:
Why not default `max_concurrency`?

Reply:
because at least on macs thats 4 cores, which means 4 processes - we can easily do more, geth doesn't so i just double the amount. if you try to do more you can crash geth in a container.

Reply:
geths idea of concurrency is different then erlangs, where you can have 100k concurrent processes on the beam, geth can have only a few. so we can't bombard the geth nodes too much. but 8 concurrent processes creating accounts is still faster than 4 (on my machine :) )